src/components/Thumbnail/Thumbnail.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC } from "react";
+import { Button } from "./Button";
+import { Description } from "./Description";
+import { Image } from "./Image";
+import { Name } from "./Name";
+
+interface IProps {
+  button: string;
+  description: string;
+  name: string;
+  icon: string;
+  alt: string;
+}
+export const Thumbnail: FC<IProps> = ({
+  button,
+  description,
+  name,
+  icon,
+  alt
+}) => (
+  <div className="p-2 w-1/5 shadow block">
+    <Image icon={icon} alt={alt} classname="mb-2" />
+    <Name children={name} />
```

Comment:
`children` are not a classic prop in this sense. we call `children` everything which is inside of the given component. check an article https://codeburst.io/a-quick-intro-to-reacts-props-children-cb3d2fce4891
this should be written as 
```
<Name>{name}</Name>
```
Same with the following lines as in `Description` and `Button`